### ISSUE TITLE
Add short names for kadalu storages CRDs

### ISSUE THREAD

**Issue by zhucan** [kadalu/kadalu#698]

**Comment by leelavg** [comment]
@zhucan it's already implemented in #609 and available from v0.8.5, do you mean anything else? you can use `k get kds` or `k get kadalu`
```
# git tag --contains e43064
0.8.5
0.8.6
0.8.6_alpha0
0.8.6_alpha1
0.8.7
```

**Comment by zhucan** [comment]
@leelavg Thanks
